src/pathfora.js

```diff
@@ -916,6 +913,39 @@
         switch (config.layout) {
         case 'modal':
         case 'slideout':
+          // Add Content recommendation if we have one
+          if (config.recommend && config.content) {
```

Comment:
should this fail or should we create the `config.content` array if it doesnt exist? if i dont set a default but i still have the recommend params we'd still want it to recommend and work right?